### DISCUSSION TITLE
Solidity 101, Section3(FundMe), Lesson 4(Math in solidity)

### DISCUSSION THREAD

**Discussion by owanemi** [Cyfrin/foundry-full-course-cu#2002] [Q&A]
function convertUsdToEth(uint256 usdAmount, uint256 ethPrice) public view returns(uint256) {
        ethPrice = getPrice();
        uint256 usdAmountToEth = (usdAmount*10e31)/ethPrice;
        return usdAmountToEth;
    }
    
    i'm trying to do the exercise in lesson 4 which is to convert usd value to eth, i'm bad with the decimal manipulation, can someone please help out

**Comment by EngrPips** [comment]
Honestly, it won't help your learning process if anyone solves it and gives you the answer, so instead, I will urge you to go through the decimal manipulation exercise here `https://piacademy.co.uk/11-plus-maths-topic/numbers/decimal-manipulation/` to improve yourself.

**Reply by owanemi** [reply]
okay thank you so much

**Reply by EngrPips** [reply]
You are welcome.
